src/Microsoft.AspNetCore.Sockets/DefaultConnectionContext.cs

```diff
@@ -22,6 +23,7 @@ public class DefaultConnectionContext : ConnectionContext,
         // This tcs exists so that multiple calls to DisposeAsync all wait asynchronously
         // on the same task
         private TaskCompletionSource<object> _disposeTcs = new TaskCompletionSource<object>();
+        internal ValueStopwatch _connectionTimer;
```

Comment:
Use a property. We generally avoid setting fields from outside the class. Keep it `internal` though.